antarest/study/storage/variantstudy/model/command/update_binding_constraint.py

```diff
@@ -79,12 +79,12 @@ def _apply(self, study_data: FileStudy) -> CommandOutput:
         updated_cfg = binding_constraints[index]
         updated_cfg.update(obj)
 
-        updated_terms = set(self.coeffs) if self.coeffs else set()
-
-        # Remove the terms not in the current update but existing in the config
-        terms_to_remove = {key for key in updated_cfg if ("%" in key or "." in key) and key not in updated_terms}
-        for term_id in terms_to_remove:
-            updated_cfg.pop(term_id, None)
+        updated_properties = self.dict(exclude={"command_context", "command_name", "version", "id"}, exclude_none=True)
```

Comment:
You should use:

```python
excluded_fields = frozenset(ICommand.__fields__)
updated_properties = self.dict(exclude=excluded_fields, exclude_none=True)
```

Reply:
I needed to add the `id` which is not in the `ICommand` fields so the code goes like this : 

```python
excluded_fields = set(ICommand.__fields__) | {"id"}
```